<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Сообщение:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbSendFile">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>Прикрепить файл</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPathToFile">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbSend">
            <property name="text">
             <string>Отправить</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listUserOnline">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Соединение</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionReconnect"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionSettings_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Установить</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionSettings_2">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>Переподключиться</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
